src/components/card/card.jsx

```diff
@@ -45,13 +45,32 @@ export default class Card extends React.Component {
 
           <div className="card-title-container align-self-end">
             <span className="card-title">{ this.props.title }</span>
+
+            <div className="card-info d-flex align-self-end justify-content-between">
+              <div className="card-heading">{ this.props.heading }</div>
+
+              <div className="card-social">
+                <div className="social-link-fb float-left">
+                  <a className="facebook-share" href={ `https://www.facebook.com/sharer/sharer.php?u=${ encodeURIComponent(this.props.facebookShareMessage) }` }
+                    target="_blank" rel="noopener noreferrer">
+                    <FacebookIcon />
+                  </a>
+                </div>
+
+                <div className="social-link-tw float-right">
+                  <a className="twitter-share" href={ `https://twitter.com/intent/tweet?status=${ encodeURIComponent(this.props.twitterShareMessage) }` } target="_blank" rel="noopener noreferrer">
+                    <TwitterIcon />
+                  </a>
+                </div>
+              </div>
+            </div>
           </div>
         </a>
 
-        <div className="card-info d-flex align-self-end justify-content-between">
-          <div className="card-heading">{ this.props.heading }</div>
+        <div className="card-info-event d-flex align-self-end justify-content-between">
+          <div className="card-heading-event">{ this.props.heading }</div>
```

Comment:
why is _-event_ being used in the `className` here for the Card` component?  Just to be clear, the scope of this PR was only for the two column layout.